claim-check/code-samples/sample-2/client-consumer/EventHubsConsumer.cs

```diff
@@ -114,10 +124,10 @@ public async Task ProcessMessages(CancellationToken cancellationToken)
             }
```

Comment:
nit:  You may want to consider calling out explicitly that this exception is expected because the `Task.Delay` operation was cancelled.   I'm not sure that it would be clear at a glance and folks may assume that the processor's `StartProcessingAsync` method blocks until cancelled, which isn't the case.